### PR TITLE
[FIX] and [ADD] version non-specific links, additional resources

### PR DESCRIPTION
**Pull request by ray-odoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,10 +153,12 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 - [Technaureus](https://www.youtube.com/channel/UChusfuYvl4Xbf7-RLP5hHFg) - English channel
 - [Odoo Bangla](https://www.youtube.com/channel/UCoVnWNoF0kQbukKqN_DxxfQ) - Bangla channel
 - [Luigys Toro](https://www.youtube.com/channel/UCAoYNRiCdqVGxAbNoiUODRg) - Spanish channel
+- [Nicholas Kosinski](https://www.youtube.com/channel/UCfmxKTk4iO2PSdSVOLKhwBA) - English channel
 
 ## Courses
 
-- [Official Odoo Courses](https://www.odoo.com/es_ES/slides/all) - Official Courses
+- [Official Odoo Courses] (https://www.odoo.com/slides) - Official Courses - English
+- [Official Odoo Courses](https://www.odoo.com/es_ES/slides/all) - Official Courses - Spanish
```
Comment (desdelinux) [inline README.md]: This link points to an English version

File: README.md [hunk 2]
```diff
@@ -19,10 +19,9 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
 ## Documentation
 
-- [User Documentation](https://www.odoo.com/documentation/14.0/applications.html) - Official user documentation
-- [Development Documentation](https://www.odoo.com/documentation/14.0/) - Official development documentation
-- [Documentation for Install and Maintain](https://www.odoo.com/documentation/14.0/administration.html) - Documentation for Install and Maintain
+- [Product Documentation](https://www.odoo.com/documentation) - Official Documentation - User Docs, Install and Maintain, Developer, Contributing
 - [Data Model Documentation for V8](http://useopenerp.com/v8) with UML diagrams.
+- [Data Model Documentation] (https://www.odoo.com/documentation) generated by ERBuilder
```
Comment (desdelinux) [inline README.md]: Where is this?

**Review by desdelinux — COMMENTED** [review]
Thank you for your contribution. You can look at the observations @ray-odoo

File: README.md [hunk 3]
```diff
@@ -19,10 +19,9 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
 ## Documentation
 
-- [User Documentation](https://www.odoo.com/documentation/14.0/applications.html) - Official user documentation
-- [Development Documentation](https://www.odoo.com/documentation/14.0/) - Official development documentation
-- [Documentation for Install and Maintain](https://www.odoo.com/documentation/14.0/administration.html) - Documentation for Install and Maintain
+- [Product Documentation](https://www.odoo.com/documentation) - Official Documentation - User Docs, Install and Maintain, Developer, Contributing
```
Comment (desdelinux) [inline README.md:22]: Is there a way to continue segmenting and targeting the latest version?

Comment (ray-odoo) [reply README.md:22]: I don't know of a way. 

With centralized documentation - does it make sense to segment it in your list?

File: README.md (hunk 2 above)
Comment (ray-odoo) [reply README.md]: Sorry, the link should be (https://soft-builder.com/en/docs/Odoo/index.html)

File: README.md (hunk 1 above)
Comment (ray-odoo) [reply README.md]: https://www.odoo.com/es_ES/slides/all shows the User Menus and other UI features (navigation particularly) in Spanish.

https://www.odoo.com/slides shows the User Menus and other UI features (navigation particularly) in English.

Video content is in English for all courses.

Comment (desdelinux) [reply README.md]: Perfect

File: README.md (hunk 2 above)
Comment (desdelinux) [reply README.md]: I find it a good resource, but it is incomplete, many tables are missing.

I recommend adding it as an additional line and keeping the one from v8

File: README.md (hunk 3 above)
Comment (desdelinux) [reply README.md:22]: Perfect, as you indicate, please update your pr with the suggested changes.

Thank you for your collaboration

File: README.md (hunk 2 above)
Comment (ray-odoo) [reply README.md]: I agree

**Comment by ray-odoo** [comment]
Done.

**Review by desdelinux — APPROVED** [review]

**Pull request merged**